### ISSUE TITLE
Shouldn't the .iml file be in .idea/?

### ISSUE THREAD

**Issue by orome** [carymrobbins/intellij-haskforce#222]
For other projects I've built using the IDE, I have the `.iml` file tucked away in the `.idea/` directory. But if I try that here my project isn't recognized. Is this normal?

**Comment by carymrobbins** [comment]
Originally in IntelliJ, `.iml` files were inside the module directory.  It seems that newer versions of IntelliJ instead put them in the `.idea/` directory.  HaskForce still currently puts them (and assumes they are) in the module directory.

What do you mean by the "project isn't recognized"?  Is there a specific error message that you receive?

**Comment by orome** [comment]
If I move it to the `.idea/` directory, the project isn't recognized as a project (e.g., no modules are recognized). This isn't the case with other projects where I do the same (move the `.iml` manually).

**Comment by carymrobbins** [comment]
Yes, HaskForce associates the `.iml` files directly with their module directories, so it expects them to be in their respective roots.  This _could_ change, but would require some work.
